src/status_im2/contexts/communities/home/view.cljs

```diff
@@ -33,38 +33,63 @@
    {:id :pending :label (i18n/label :t/pending) :accessibility-label :pending-tab}
    {:id :opened :label (i18n/label :t/opened) :accessibility-label :opened-tab}])
 
+(defn empty-state
+  []
+  [rn/view {:style style/empty-state-container}
+   [rn/view {:style style/empty-state-placeholder}]
+   [quo/text
+    {:accessibility-label :communities-rule-index
+     :weight              :semi-bold
+     :size                :paragraph-1}
+    (i18n/label :t/no-communities)]
+   [quo/text
+    {:accessibility-label :communities-rule-index
+     :weight              :regular
+     :size                :paragraph-2}
+    (i18n/label :t/no-communities-sub-title)]])
+
 (defn home
   []
   (let [selected-tab (reagent/atom :joined)]
     (fn []
       (let [{:keys [joined pending opened]} (rf/sub [:communities/grouped-by-status])
+            {:keys [public-key key-uid]}    (rf/sub [:multiaccount])
+            photo-path                      (rf/sub [:chats/photo-path public-key])
+            customization-color             (rf/sub [:profile/customization-color key-uid])
             selected-items                  (case @selected-tab
                                               :joined  joined
                                               :pending pending
                                               :opened  opened)
             top                             (safe-area/get-top)]
         [:<>
-         [rn/flat-list
-          {:key-fn                            :id
-           :content-inset-adjustment-behavior :never
-           :header                            [rn/view {:height (+ 245 top)}]
-           :render-fn                         item-render
-           :data                              selected-items}]
+         (if (empty? selected-items)
+           [empty-state]
+           [rn/flat-list
+            {:key-fn                            :id
+             :content-inset-adjustment-behavior :never
+             :header                            [rn/view {:height (+ 245 top)}]
```

Comment:
should pass styles to `:style` prop, also maybe you can define the header view in a separate fn